workspaces/ui/src/entrypoints/index.js

```diff
@@ -1,15 +1,20 @@
 import React from 'react';
-import {Route, Switch} from 'react-router-dom';
+import { Route, Switch } from 'react-router-dom';
 import Development from './development';
 import LocalCli from './localcli';
 import WelcomePage from '../components/support/WelcomePage';
 
 export default function TopLevelRoutes() {
   return (
     <Switch>
+      <Route
```

Comment:
I think the path that would have fewer conditionals and allow greater composition would be to fork the Development component into a PrivateSession/CustomerSession component.

Reply:
which we would then mount here as ```<Route strict path="/testing/private-examples/:sessionId" component={PrivateExampleRoot} />```

Reply:
I considered this, but decided adding another Entrypoint like `Development` would be more confusing than this 1 additional parameter is worth, especially as there is only a single parameter as of yet (session id). Let the private session entrypoint prove the need for it's existence, that it's truly some novel concept and we can easily change this.